src/app/cluster/details/cluster/edit-cluster/style.scss

```diff
@@ -15,6 +15,7 @@
 @use 'variables';
 
 .edit-cluster-form {
+  height: 60vh;
```

Comment:
Instead of changing the height I would consider changing max-height instead, so it will prevent the window being too big but also keep the current behavior.